ethtools/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+description = "Ethcore Ethereum tools"
+homepage = "http://ethcore.io"
+license = "GPL-3.0"
+name = "ethtools"
```

Comment:
If, say, we decide not import geth keys anymore, we just delete one mod from ethtools and one call from command line options

and if we decide not to use separate crate for it, we will have to refactor our main crate and try to delete all this stuff and not to break anything and not to leave anything useless